src/aiidalab_qe/app/configuration/advanced.py

```diff
@@ -261,6 +265,53 @@ def reset(self):
             # reset the override checkbox
             self.override.value = False
 
+    def _display_mesh(self, _=None):
+        if self.input_structure is not None:
```

Comment:
```suggestion
        if self.input_structure is None:
        return
```
Then the left part of code in this function has no need to be indented.